README.md

```diff
@@ -1,36 +1,77 @@
-apt install git curl tmux zsh vim-nox bat ncdu ranger neovim python3 python3-pip htop neofetch powerline ifstat nethogs exa
 
-pip3 install --user git+https://github.com/powerline/powerline
+> Run [script.sh](https://github.com/yashensha/dotfiles/blob/main/script.sh) or Execute them seperately. 
```

Comment:
edit the blob url to ./script.sh

Reply:
Merge them.

Reply:
i asked you to change the absolute url of file to relate url. then it can be pointed to the same repo

Reply:
how can i relate link from save repo unless that repo don't have that file?
i don't know how

Reply:
so i removed that link

Reply:
yep i understand now

Reply:
changes made to readme file